beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -137,11 +138,22 @@ def main():
         try:
             job = bs.reserve()
             job_data = json.loads(job.body)
+
+            debug_logs_file = os.path.join(
+                job_data["logs_dir"],
+                "service_debug.log"
```

Comment:
I agree, linter and workers are not yet streamlined. so it would be hard to put them in a central point